docs/section12/Rule12-3.md

```diff
@@ -1,38 +1,58 @@
-
 # Receptacle - Rule 12-3
-
+**Schema Version** 0.0.37  
+**Primary Rule:** True  
 **Rule ID:** 12-3  
-**Rule Description:** User RMR Space ID in Proposed RMR?  
-**Rule Assertion:** Proposed RMR = User RMR  
-**Appendix G Section:** Section Table G3.1-12 Receptacles: Modeling Requirements for the Proposed design  
+**Rule Description:** When receptacle controls installed in spaces where not required by Standard 90.1 2019, Section 8.4.2 are included in the proposed building design, the hourly receptacle shall be reduced as specified in Standard 90.1-2019, Table G3.1 Section 12, Proposed Building Performance.   
+**Appendix G Section:** Table G3.1 Section 12 Baseline Building Performance  
 **Appendix G Section Reference:** None  
 
-**Applicability:** All required data elements exist for P_RMR and U_RMR  
-**Applicability Checks:**  
-
-  1. User RMR Space Count > 0  
+**Applicability:** All required data elements exist for B_RMR and P_RMR  
+**Applicability Checks:**
+1. Receptacle controls are installed in spaces where not required by Standard 90.1 2019, Section 8.4.2  
 
 **Manual Check:** None  
-**Evaluation Context:** Each Data Element  
+**Evaluation Context:** Each Miscellaneous Equipment object in the proposed model    
 **Data Lookup:** None  
+**Function Call:**
+1) get_component_by_id
+2) compare_schedules
+
+## Applicability Checks:  
+- Create a list of the lighting space types that correspond to space types that may have receptacle control requirements in Section 8.4.1: `EXPECTED_RECEPTACLE_CONTROL_SPACE_TYPES = ["OFFICE_ENCLOSED", "CONFERENCE_MEETING_MULTIPURPOSE_ROOM", "COPY_PRINT_ROOM", "LOUNGE_BREAKROOM_HEALTH_CARE_FACILITY", "LOUNGE_BREAKROOM_ALL_OTHERS", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_PENITENTIARY", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_SCHOOL", "CLASSROOM_LECTURE_HALL_TRAINING_ROOM_ALL_OTHER", "OFFICE_OPEN_PLAN"]`  
```

Comment:
@JacksonJ-KC 8.4.1 is voltage drop, i assume you mean 8.4.2?

Reply:
should this statement-
Create a list of the lighting space types that correspond to space types that may have receptacle control requirements in Section 8.4.1: 
be
Create a list of the lighting space types that correspond to space types that are required to have automatic receptacle control as per Section 8.4.2:

Reply:
We are not able to say for sure that there needs to be receptacle controls because we don't know if the receptacles are 125 V, 15 or 20 amp

> At least 50% of all 125 V, 15 and 20 amp receptacles in all private offices, conference
rooms, rooms used primarily for printing and/or copying functions, break rooms,
classrooms, and individual workstations